### ISSUE TITLE
About Training Settings

### ISSUE THREAD

**Issue by hongxiaoy** [bhyang/diffusion-es#2]
Thanks for you excellent work! 

I wonder the detail of training settings, like the number of gpus you used, and the batch size of the data and learning rate.

Can you help me with these parameters, thanks~

**Comment by bhyang** [comment]
Model training details can be found in the appendix [here](https://arxiv.org/pdf/2402.06559#page=13). We used batch size 256, learning rate 1e-4, and a single GPU.
